docs/reference-guides/user-settings/user-preferences.md

```diff
@@ -2,16 +2,59 @@
 title: User Preferences
 ---
 
-Each user can choose preferences to personalize their Rancher experience. To change preference settings, open the **User Settings** menu and then select **Preferences**.
+Each user can choose preferences to personalize their Rancher experience. To change preference settings:
+
+1. Click on your user avatar in the upper right corner.
+1. Click **Preferences**.
+
+## Language
+
+Choose your language for the Rancher UI. Options include:
+
+- English
+- 简体中文 (Simplified Chinese)
 
 ## Theme
 
 Choose your background color for the Rancher UI. If you choose **Auto**, the background color changes from light to dark at 6 PM, and then changes back at 6 AM.
 
-## My Account
+## Login Landing Page
+
+Choose the page displayed after logging in. Options include:
+
+- The home page
+- The area you last visited
+- A specific cluster of your choosing
+
+## Display Settings
+
+Choose the way certain information is displayed. The following can be configured:
+
+- Date format
+- Time format
+- Table rows per page
+- Number of clusters to show in side menu
+
+## Confirmation Setting
```

Comment:
```suggestion
## Confirmation Settings
```
Only one so far, but the plural matches the other headings and there might be more in the future